Dockerfile

```diff
@@ -35,3 +35,5 @@ run chmod +x /tmp/install-cmake && /tmp/install-cmake && rm /tmp/install-cmake
 # Coverage reporting.
 copy bin/install-lcov /tmp/install-lcov
 run chmod +x /tmp/install-lcov && /tmp/install-lcov && rm /tmp/install-lcov
+
+run apt-get clean -y
```

Comment:
This had me googling. It's new to me.

Have a look:

- https://gist.github.com/marvell/7c812736565928e602c4
- https://stackoverflow.com/questions/61990329/benefits-of-repeated-apt-cache-cleans
- https://docs.docker.com/develop/develop-images/instructions/#apt-get

I recommend we omit the call to `apt-get clean`, as well as omit any `rm`. I doubt it saves much space, and it's not clear which incantation is "best."

Reply:
ok

Reply:
Fixed in 8769e8cf7c359e45f8cdbb8a80a36bc8a443634f